cmd/launcher/launcher.go

```diff
@@ -210,7 +211,11 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 	runGroup := rungroup.NewRunGroup(logger)
 
 	// Add the log checkpoints to the rungroup, and run it once early, to try to get data into the logs.
-	checkpointer := checkups.NewCheckupLogger(logger, k)
+	checkpointLogger := log.With(logutil.NewServerLogger(true))
+	checkpointer := checkups.NewCheckupLogger(
+		teelogger.New(checkpointLogger, locallogger.NewKitLogger(filepath.Join(opts.RootDirectory, "debug.json"))),
```

Comment:
I think Zack is doing this to ensure that we have a json logger that gets passed to the checkpointer?

I think we should come at this slightly differently:
* I'd be okay assuming most usage uses a json logging and not angsting about errors in the fmt logger
* We could swap the command line usage to use json logging instead of fmt logging
* We could attempt to detect and then??? marshal differently? (This sounds like a lot of work)

Reply:
pushed, thank you!